public/stylesheets/detailed-results.css

```diff
@@ -104,27 +104,81 @@ body {
 
 #game-info-container {
   width: 100%;
+  min-height: 250px;
+  background-color: #333;
 }
 .detailed-results-text {
   flex-direction: column;
-  background-color: #333;
+  overflow-wrap: break-word;
+}
+.game-details-item {
+  font-size: 20px;
+  margin-bottom: 15px;
+}
+.details-text,
+.game-website {
+  color: white;
+  display: inline;
+  margin-left: 10px;
 }
 .game-name {
-  font-size: 25px;
+  font-size: 35px;
   text-align: center;
   margin-bottom: 20px;
+  width: 80%;
+  align-self: center;
 }
 .game-details {
   font-size: 18px;
   flex-grow: 1;
   margin-left: 28px;
 }
+.store-icons {
+  height: 40px;
+  width: auto;
+  margin-left: 15px;
+  margin-bottom: -12px;
+}
+.meta-score-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-self: end;
+  margin: 0px 30px 20px 0px;
```

Comment:
I think I mentioned it long time ago but you should a padding on the card wrapper rather than giving each element their separate spacing styles. If you do that, you'll fix another issue which is the fact that spacing is not unified across the card:
![image](https://github.com/DmitrijSidorkin/pixel-thing/assets/9388169/64300f33-0ad0-4125-8a0b-f672b95bdd08)

Reply:
Did try to change where i put the margins a bit, this specific line for meta score did not really change though since that placement was kinda intentional, unless i am missing something here.

Reply:
Well the spacing is all over the place imo, the text is touching the side of the card, like overall there should be unified padding in the card in general. If you look at the meta score element then you see it has its own padding to the right (why tho)

Reply:
Felt like meta-score being pushed a bit to the right was visually more appealing :d

Reply:
I agree about meta data, but the text touching the card border is a bit eh. At the end of the day we don't really have a design so kinda have to eyeball everything, and so this was my general impression of how the page looks.